Core/MemMap.cpp

```diff
@@ -314,6 +314,54 @@ void Reinit() {
 	Core_NotifyLifecycle(CoreLifecycle::MEMORY_REINITED);
 }
 
+static void CopyAlignedFast(uint8_t *dst8, const uint8_t *src8, int l, int h) {
+	// All threads round both down, so they will still align.
+	static const int stride = 0x40;
+	l &= ~(stride - 1);
+	h &= ~(stride - 1);
+	const int n = h - l;
+	const uint8_t *src = (const uint8_t *)(src8 + l);
+	uint8_t *dst = (uint8_t *)(dst8 + l);
+
+	for (int i = 0; i < n; i += stride) {
+		memcpy(dst, src, stride);
+		src += stride;
+		dst += stride;
+	}
```

Comment:
Not sure I understand - how is this loop faster than a large memcpy from l to h?

Reply:
Is it different with `memcpy(dst, src, n*stride);` ? or may be it was used to catch exception occurred on a certain stride?

Reply:
Can be wrong, but it looks like this code is automatically optimized into what this commit replaced(https://github.com/hrydgard/ppsspp/pull/14372/commits/733bd927c8f213316173c760578328e81d1f5a76) where possible, keeping the thing simpler and easier to read, but if so a comment would be nice for the future.

Reply:
Yeah, at first I replaced it with SSE and it was faster.  Using this loop kept the speed improvement.

It didn't seem to make any difference really on clang, it was just helping MSVC.

-[Unknown]

Reply:
I think I would prefer the simpler code unless there's a large speed difference on MSVC...

Reply:
Okay, it added another 1ms (8.5ms -> 7.5ms) as noted in my pull request description originally, but I can pull it out.  8.5 is already pretty good.

It does vary a bit too, that's the average.  It was definitely faster with this, which was of course disappointing.  Maybe keeping the aligning of h and l will still make one memcpy faster.

-[Unknown]

Reply:
Confirmed that `memcpy(dst8 + l, src8 + l, h - l);` (even keeping the alignment of `l` and `h`) loses the speed gain.

`memmove()` is a bit more than halfway between, so let's go with that.

-[Unknown]